Apps/LabRecorder/cmake/FindLSL.cmake

```diff
@@ -1,74 +0,0 @@
-# - Try to find the labstreaminglayer library
```

Comment:
I removed this for once because CMake autogenerates the necessary information with exported targets.